src/openforms/config/data.py

```diff
@@ -10,6 +10,7 @@ class Entry:
     status: Optional[bool] = None
     status_message: str = ""
     actions: Optional[List[Tuple[str, str]]] = None
+    error: Optional[str] = None
```

Comment:
```suggestion
    error: str = ""
```
`bool(error)` in both situation is `False` for the default value.

Improvement: keep types to check for "empty" value simple.